share/spice/open_nic/open_nic.css

```diff
@@ -0,0 +1,64 @@
+/*
+    Document   : zeroclickinfo-spice/share/spice/open_nic/open_nic.css
```

Comment:
You can remove these comments, Codio happens to spit this out, but we don't really need/use it.